docs/guide/tutorial-i18n.md

```diff
@@ -41,8 +50,8 @@ return [
 ```
 
 The default value for the [[yii\base\Application::$sourceLanguage|source language]] is `en-US`, meaning
-US English. It is recommended that you keep this default value unchanged, because it is usually much easier
-to find people who can translate from English to other languages than from non-English to non-English.
+US English. It is **recommended** that you keep this default value unchanged. Usually it is much easier
+to find people who can translate from 'English to other languages' than from 'non-English to non-English'.
```

Comment:
to improve readability. The terminology look confusing (and perhabs even wrong). Therefore  adding quotes are useful. Its also helps easier comparing.